cel/env_test.go

```diff
@@ -271,6 +271,22 @@ func TestLibraries(t *testing.T) {
 	}
 }
 
+func TestFunctions(t *testing.T) {
+	e, err := NewEnv(OptionalTypes())
+	if err != nil {
+		t.Fatalf("NewEnv() failed: %v", err)
+	}
+	for _, expected := range []string{"optional.of", "or"} {
```

Comment:
You could use the list of the optional functions here, but I would probably also recommend adding a test which covers a function library that has a version attached to it, possibly updating the `ext/strings.go` test to validate the same feature and assert that a function does not exist when an older version is pinned?

Reply:
This is a good idea. I've added a test that functions as a backward compatibility linter for `ext/strings.go`.